isort/finders.py

```diff
@@ -35,6 +35,11 @@
 except ImportError:
     Pipfile = None
 
+try:
+    from functools import lru_cache
+except ImportError:
```

Comment:
The `develop` branch dopped support for EOL Pythons. So maybe this should target that branch and then we wouldn't need to includ the `backports` dependency.

Reply:
I specifically targeted this branch in order to get a fix out in the 4.3.x series which still support Python 2. The fix can easily be ported to the develop branch afterwards.

Reply:
Gotcha. Sounds good to me.